tests/unit/test_vmware_exporter.py

```diff
@@ -225,9 +225,14 @@ def test_collect_vm_perf():
         'disk.usage.average': 4,
         'disk.read.average': 5,
         'disk.write.average': 6,
-        'mem.usage.average': 7,
-        'net.received.average': 8,
-        'net.transmitted.average': 9,
```

Comment:
Changing this from `9` to `14` is what broke the test - either don't change the old values or change line 206 to match the new value:

```
    metric_1 = mock.Mock()
    metric_1.id.counterId = 14
    metric_1.value = [9]
```